keyboards/acheron/austin/keymaps/wk/keymap.c

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
I would prefer it if this keymap were removed entirely, since it's identical to the default.

Reply:
However denoted, the GB runner insisted that those keymaps be kept due to an ease to understand for the end user.

Now both WK and WKL layouts were removed and only default remains. All three are the same.